### PR TITLE
DOC-3222: Added new Add Replication UI and descriptions of compressio…

### PR DESCRIPTION
**Pull request by tonyjhillman**
…n and network usage limit

### CODE REVIEW

**Comment by tonyjhillman** [comment]
Is my description of XDCR Network usage Limit (in the section XDCR Advanced Settings) accurate?

File: content/xdcr/xdcr-create.dita [hunk 1]
```diff
@@ -122,13 +134,37 @@
...
+        <li>
+          Select an <uicontrol>XDCR Protocol</uicontrol>. The options are:
+          
+          <ul>
+            
+            <li>Version 1. This uses the REST protocol. If the Elasticsearch plug-in is being used,
+            choose Version 1.
+            </li>
+            
+            <li>Version 2. This is the default. It uses memcached REST protocol for replication, a high-performance
```
Comment (JFlath) [inline content/xdcr/xdcr-create.dita]: I know it's text carried forward, but memcached doesn't have a REST protocol, it should probably read:

>It uses the Memcached Binary Protocol...

Comment (JFlath) [inline content/xdcr/xdcr-create.dita]: I know it's text carried forward, but memcached doesn't have a REST protocol, it should probably read:

>It uses the Memcached Binary Protocol...

File: content/xdcr/xdcr-create.dita [hunk 2]
```diff
@@ -122,13 +134,37 @@
...
+              mode that directly uses the memcached protocol on the destination nodes.
+              Choose version 2 when setting up a new replication with Couchbase Server 2.2
```
Comment (JFlath) [inline content/xdcr/xdcr-create.dita]: Given that we mention 2.2 (quite the antique!), should this be updated? AFAIK using Version 1 for anything other than ES is deprecated now

File: content/xdcr/xdcr-create.dita [hunk 3]
```diff
@@ -253,6 +286,11 @@
                 <entry>XDCR Statistics Collection Interval</entry>
                 <entry>Shows how often XDCR Statistics is updated.</entry>
               </row>
+              <row>
+                <entry>XDCR Network Usage Limit (MB/sec)</entry>
+                <entry>The upper limit for network usage during replication, per node, specified in megabytes per
```
Comment (JFlath) [inline content/xdcr/xdcr-create.dita]: (nitpick) Could we make it `per source node`?

**Pull request merged**